OrbitCore/FunctionFramePointerValidator.cpp

```diff
@@ -8,6 +8,8 @@
 
 #include <iostream>
 
+#include <OrbitBase/Logging.h>
```

Comment:
Let's be more consistent and use "" for Orbit includes.

Reply:
Fixed, thank you!